File: LeetCodeCPlusPlus/py70.cpp
// 爬楼梯，可以爬一级或两级，n级台阶共有多少种
class Solution {
public:
    int climbStairs(int n) {
        int ppre = 0;
        int pre = 1;
        int cur = 0;
        for (int i = 0; i<n; i++) {
            cur = ppre + pre;
            ppre = pre;
            pre = cur;
        }
        return cur;
    }
};

File: LeetCodePython/py70.py
# 爬楼梯，可以爬一级或两级，n级台阶共有多少种
class Solution:
    def climbStairs(self, n: int) -> int:
        pre = 1
        ppre = 0
        cur = 0
        for i in range(n):
            cur = pre + ppre
            ppre = pre
            pre = cur
        
        return cur